src/backend/distributed/sql/downgrades/citus--10.0-4--10.0-3.sql

```diff
@@ -0,0 +1,9 @@
+--
```

Comment:
why do we put this file under "downgrades" folder? We didn't put the other ones there

Reply:
this is the last version of the 'backport'.

This one will be merged before we release 10.1, so we don't need this file in de normal folder. Instead we upgrade directly from 10.0-4--10.1-1.

This is not possible with the actual backports, however due to 10.1 being blocked by this release we can follow the normal upgrade route.

eg. the issue is fixed in 10.0.4 and 10.1 is released thereafter.

Reply:
a subtile thing that is important, we do want this fix from 10.0-3--10.0-4 to be on the normal upgrade path. That will make sure that any future version of Citus, either cleanly installed or upgraded, will pass this upgrade path.